src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -31,7 +30,11 @@ class EditIssuePage(
 
     fun fillForm(): EditIssuePage {
         summaryLocator.clearAndTypeIfPresent("summary")
-        descriptionLocator.clearAndTypeIfPresent("description")
+        val descriptionFieldId = "description"
+        if(driver.isElementPresent(By.id(descriptionFieldId))){
+            RichTextEditorTextArea(driver, driver.findElement(By.id(descriptionFieldId)))
+                .overwriteIfPresent("description")
```

Comment:
Is the "description" here the same as `descriptionFieldId`. If not, will it work when Jira's language is other than English?

Reply:
It's the value to be sent to Description field. In general, English text shall be accepted regardless the language set.